README.md

```diff
@@ -36,9 +36,11 @@ Jest matcher that performs image comparisons using [pixelmatch](https://github.c
 `toMatchImageSnapshot()` takes an optional options object with the following properties:
 
 * `customDiffConfig`: Custom config passed [pixelmatch](https://github.com/mapbox/pixelmatch#pixelmatchimg1-img2-output-width-height-options) (See options section)
-  * By default we have set the threshold to 0.01, you can increase that value by passing a customDiffConfig as demonstrated below
+  * By default we have set the `threshold` to 0.01, you can increase that value by passing a customDiffConfig as demonstrated below. Please note the threshold set in the customDiffConfig is the per pixel sensitivity threshold. For example with a source pixel colour of #ffffff (white) and a comparison pixel colour of #fcfcfc (really light grey) if you set the threshold to 0 then it would trigger a failure *on that pixel*. However if you were to use say 0.5 then it wouldn't, the colour difference would need to be much more extreme to trigger a failure on that pixel, say #000000 (black)
```

Comment:
Can you break up this wall of text into multiple bullets? And wrap the hex values in single ticks (`)?

Reply:
Done